scripts/validate_project.sh

```
@@ -0,0 +1,310 @@
+fi
+
+# 9. 로그 검증 (오류 확인)
+echo -e "\n${BLUE}📋 9. 서비스 로그 검증${NC}"
+
+check_logs_for_errors() {
+    local service=$1
+    local error_patterns=("ERROR" "FATAL" "Exception" "Traceback" "failed" "FAILED")
+    
+    log_info "${service} 로그에서 오류 검색 중..."
+    
+    local logs=$(docker-compose logs $service 2>/dev/null || echo "")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)